python/module.cpp

```diff
@@ -113,9 +113,12 @@ PYBIND11_MODULE(singularity_eos, m) {
 #endif
 
 #ifdef SINGULARITY_USE_EOSPAC
-  eos_class<EOSPAC>(m, "EOSPAC")
+  eos_class<EOSPAC, true>(m, "EOSPAC")
```

Comment:
I assume it's the `use_scratch` template parameter below, right? Would it make sense to add a comment or just introduce a `bool eospac_use_scratch = true` variable to explain this since it deviates from the construction of the other classes?

Reply:
Fixed.